modules/microvm/virtualization/microvm/loki-local-config.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+auth_enabled: false
+
+server:
+  http_listen_port: 3100
+  grpc_listen_port: 9096
+
+common:
+  instance_addr: 127.0.0.1
+  path_prefix: /tmp/loki
```

Comment:
I would probably still put it in `/var/lib/loki` also this VM doesn't realistically has any other services /tmp is still world-readable after all. In my experience people might end up copying this at some point to a place where this matters.

Reply:
Sorry, prev comment still applies structurally but for `alloy`, not for `promtail` - I should be more focused but I wasn't. My excuse is Friday and the heat making my brain soft ...

Reply:
Hi @vilvo  I have addressed your review comments in https://github.com/tiiuae/ghaf/pull/625
Please review it.